cluster/olm/bundle/manifests/rbac-manager.serviceaccount.yaml

```diff
@@ -1,5 +1,7 @@
 # Generated by https://github.com/upbound/olm-bundle
 apiVersion: v1
+imagePullSecrets:
```

Comment:
Does this work?

Reply:
Do you mean if there is no such secret?

Reply:
I remember it works when defined as part of pod spec, so I would expect same here.

Reply:
By the way, it was already like that for Crossplane service account: https://github.com/upbound/universal-crossplane/blob/a94877acff85cd7292557599998578924688ba21/cluster/olm/bundle/manifests/crossplane.serviceaccount.yaml#L4

Seems that same introduced recently for rbac.https://github.com/crossplane/crossplane/commit/0a39ad716d14fff92125440c58a50d565278f527

Reply:
Hmm I thought it was typo or something because it's top-level, but it seems it's part of `ServiceAccount` spec https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/service-account-v1/#ServiceAccount